src/test/scala/calc/RunTest.scala

```diff
@@ -18,7 +19,7 @@ import org.scalajs.jsenv.JSConsole
  *  result.
  */
 class RunTest {
-
+  val r = Random
```

Comment:
Consider seeding your random to get deterministic tests (not really relevant here I know, but in general very relevant).